tools/lsp/language/completion.rs

```diff
@@ -573,7 +573,9 @@ fn add_components_to_import(
     let mut last = 0u32;
     for import in current_doc.ImportSpecifier() {
         if let Some((loc, file)) = import.ImportIdentifierList().and_then(|list| {
-            let id = list.ImportIdentifier().last()?;
+            let node = list.ImportIdentifier().last()?;
+            let id = crate::util::last_non_ws_token(&node)
+                .unwrap_or_else(|| node.first_token().unwrap());
```

Comment:
Are you sure there is always a first_token there?
Maybe this would be safer with

```suggestion
                .or_else(|| node.first_token())?;
```

That way even if we refgactor the pasrer to recover more case, we wouldn't panic.